src/ape/managers/project.py

```diff
@@ -1515,7 +1516,43 @@ def from_manifest(
         Returns:
             :class:`~ape.managers.project.ProjectManifest`
         """
-        return Project.from_manifest(manifest, config_override=config_override)
+        config_override = config_override or {}
+        manifest = _load_manifest(manifest) if isinstance(manifest, (Path, str)) else manifest
+        return Project(manifest, config_override=config_override)
+
+    @classmethod
+    def from_python_library(
+        cls, package_name: str, config_override: Optional[dict] = None
+    ) -> "LocalProject":
+        """
+        Create an Ape project instance from an installed Python package.
+        This is useful for when Ape or Vyper projects are published to
+        pypi.
+
+        Args:
+            package_name (str): The name of the package's folder that would
+              appear in site-packages.
+            config_override (dict | None): Optionally override the configuration
+              for this project.
+
+        Returns:
+            :class:`~ape.managers.project.LocalProject`
+        """
+        for site_packages_path_str in getsitepackages():
```

Comment:
I feel like `importlib.resources` may have better API for this

Reply:
OK it has been refactored to using `importlib.resources` instead.